pkg/ipmi/ipmi.go

```diff
@@ -458,3 +463,44 @@ func (i *IPMI) GetSELInfo() (*SELInfo, error) {
 	}
 	return &info, nil
 }
+
+func (i *IPMI) GetLanConfig(channel byte, param byte) ([]byte, error) {
+	req := &req{}
+	req.msg.netfn = _IPMI_NETFN_TRANSPORT
+	req.msg.cmd = _BMC_GET_LAN_CONFIG
+
+	var data [4]byte
+	data[0] = channel
+	data[1] = param
+	data[2] = 0
+	data[3] = 0
+	req.msg.data = unsafe.Pointer(&data)
```

Comment:
This won't do the right thing I think. usually you do this: unsafe.Pointer(&data[0])

see the Go source, e.g., go/src/syscall/syscall_linux.go, for examples.

Reply:
Modified unsafe.Pointer(&data[0])